hazelcast/src/main/java/com/hazelcast/multimap/impl/operations/MultiMapOperationFactory.java

```diff
@@ -24,8 +25,7 @@
 
 import java.io.IOException;
 
-public class MultiMapOperationFactory
-        implements OperationFactory {
```

Comment:
That's a premature optimization. There's just one Operation factory for multimap.